vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/AllureReportGenerator.java

```diff
@@ -100,15 +111,30 @@ private void generateReport()
         LOGGER.info("Allure report is successfully generated at: {}", reportDirectory.getAbsolutePath());
     }
 
+    private void writeExecutorInfo() throws IOException
+    {
+        ExecutorInfo executorInfo = propertyMapper.readValue("allure.executor.", ExecutorInfo.class);
+
+        try (BufferedWriter writer = Files.newBufferedWriter(resultsDirectory.toPath().resolve("executor.json")))
```

Comment:
is the executor data reflected in history json for each run in order to keep mapping between runs and executors?

Reply:
the part of executor info will appear in trends